src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj

```diff
@@ -174,6 +174,12 @@
     <None Update="SessionRecords\Test.Azure.Management.Logic.WorkflowsScenarioTests\ValidateWorkflow.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SessionRecords\Test.Azure.Management.Logic.WorkflowsScenarioTests\ListSwagger.json">
```

Comment:
@shimedh no need to add individual files.
Rather we should get the entire directory contents
<None Update="SessionRecords\**\*.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>    
    </None>